flexmeasures/api/v3_0/assets.py

```diff
@@ -185,6 +187,112 @@ def index(
 
         return response, 200
 
+    @route(
+        "/<id>/sensors",
+        methods=["GET"],
+    )
+    @use_kwargs(
+        {
+            "asset": AssetIdField(data_key="id"),
+        },
+        location="path",
+    )
+    @use_kwargs(
+        {
+            "page": fields.Int(
+                required=False, validate=validate.Range(min=1), default=1
+            ),
+            "per_page": fields.Int(
+                required=False, validate=validate.Range(min=1), default=10
+            ),
+            "filter": SearchFilterField(required=False, default=None),
+            "unit": fields.Str(required=False, default=None),
+        },
+        location="query",
+    )
+    @as_json
+    def asset_sensors(
+        self,
+        id: int,
+        asset: GenericAsset | None,
+        page: int | None = None,
+        per_page: int | None = None,
+        filter: list[str] | None = None,
```

Comment:
As I wrote earlier, we can make this endpoint wimpker - no search filter, no unit param. Short and sweet. The index one has bells and whistles